spec/request/preneeds/application_forms_request_spec.rb

```diff
@@ -3,6 +3,7 @@
 
 RSpec.describe 'Preneeds Application Form Integration', type: :request do
   include SchemaMatchers
+  before(:each) { Redis.current.flushall }
```

Comment:
I hope this doesn't break other specs because of the order they run in. It shouldn't, but something to watch out for in case it does.